nav2_map_server/include/nav2_map_server/map_saver.hpp

```diff
@@ -37,13 +39,14 @@ class MapSaver : public nav2_util::LifecycleNode
 public:
   /**
    * @brief Constructor for the nav2_map_server::MapSaver
+   * @param is_pcd Bool for distinguishing b/w pcd and image, false by default
```

Comment:
Initially, the parameter was there to serve as a switch for 3D services but later I found that this could be avoided so removed it, Once the design and other changes are finalized, I will fix this docstring too.